CommandLine/testing/SimpleTests/math_test.sog/create.edl

```diff
@@ -0,0 +1,23 @@
+float f = (float) -.5;
+double d = -.5;
+// long double ld = -.5;
+int i = -30;
+unsigned u = 30;
+long l = -60;
+unsigned lu = 60;
+variant v = -120;
+var a = -150;
+
+cons_show_message("Test start!");
+gtest_assert_eq(abs(f),  -f);
```

Comment:
I'm no seeing anything about the tests between the start / ens mesages in the log